pkg/Microsoft.Private.CoreFx.UAP/Configurations.props

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      uap-Windows_NT;
-      uapaot-Windows_NT;
+      uap;
```

Comment:
But if we do this, won't we try to build this package in non-Windows builds as well? I believe that would fail since most uap configurations are Windows only.